.github/workflows/close_pylon_issue.yml

```diff
@@ -3,16 +3,13 @@ name: Close Pylon Ticket on Issue or Pull Request Closure
 on:
   issues:
     types: [closed]
-  pull_request:
+  pull_request_target:
```

Comment:
The Pylon actions don't actually need an environment, since they don't need the PR code, just the PR details.
To be safe I actually removed the checkout which is unneeded.